book.rb

```diff
@@ -0,0 +1,14 @@
+class Book
+  attr_reader :rentals
```

Comment:
- [ ] I think it would be great if you made the rentals to be `attr_accessor` instead of `attr_reader` because we need to get and set it here to make the required relationship bwteen it and the book 💪🏻🔥💯